MPF.Core/Modules/DiscImageCreator/Parameters.cs

```diff
@@ -496,7 +496,18 @@ public override void GenerateSubmissionInfo(SubmissionInfo info, Options options
                             case RedumpSystem.SonyPlayStation3:
                             case RedumpSystem.SonyPlayStation4:
                             case RedumpSystem.SonyPlayStation5:
-                                trimLength = 264;
+                                if (info.SizeAndChecksums!.Layerbreak3 != default)
+                                {
+                                    trimLength = 520;
+                                }
+                                else if (info.SizeAndChecksums!.Layerbreak2 != default)
+                                {
+                                    trimLength = 392;
+                                }
+                                else
+                                {
+                                    trimLength = 264;
+                                }
```

Comment:
Minor, but for each of these if/else trees, can you remove the `{` and `}`?

Reply:
fixed (it is my practice)